src/Proxy/AbstractProxy.php

```diff
@@ -109,6 +109,10 @@ protected function getParameterCode(ReflectionParameter $parameter): string
             $nullablePrefix = (PHP_VERSION_ID >= 70100 && $reflectionType->allowsNull()) ? '?' : '';
             $nsPrefix       = $reflectionType->isBuiltin() ? '' : '\\';
             $type           = $nullablePrefix . $nsPrefix . (string) $reflectionType;
+            // prevent double "\\" prefix for global namespace type hints:
```

Comment:
@lisachenko Other option would be to check for http://php.net/manual/en/reflectionclass.isinternal.php on line 110 as well. Kinda don't like string manipulation here...

Reply:
I don't like the dump string manipulation either, but am not really understanding in which cases we need to add the `\` ..
Could you show how this could be done?

Reply:
Maybe just add `ltrim()` code here to prevent duplication?

Reply:
Ltrim FTW !